src/app/components/todolist/todolist.component.ts

```diff
@@ -1,24 +1,40 @@
 import { Component, OnInit } from '@angular/core';
 import { fromService } from "../../services/form.service";
+
 @Component({
   selector: "app-todolist",
   templateUrl: "./todolist.component.html",
   styleUrls: ["./todolist.component.css"]
 })
 export class TodolistComponent implements OnInit {
   list: any;
-  selectedTask = { name: "",status:false };
+  selectedTask = { name: "", Date: "", status: false };
   editable: boolean;
-  
-
-  constructor(private formService: fromService) {}
+  currentdate = new Date();
+   date: any;
+  constructor(private formService: fromService) {
+    const year = this.currentdate.getFullYear();
```

Comment:
move this year month date function to a common service .
And call function from service.